cmd/loop/quote.go

```diff
@@ -132,6 +133,68 @@ func quoteOut(ctx *cli.Context) error {
 		return err
 	}
 
-	printRespJSON(quoteResp)
+	printQuoteOutResp(quoteReq, quoteResp, ctx.Bool("verbose"))
 	return nil
 }
+
+func printQuoteInResp(req *looprpc.QuoteRequest,
```

Comment:
I think the current print out is very nice and this is just a suggestion/idea. Maybe we could also use `text/tabwriter` (https://golang.org/pkg/text/tabwriter/) to align the numbers in the rightmost column to the right? Could improve readability as it is more natural I believe. WDYT?

Reply:
Yeah it's better to align the numbers to the right so it's easier to do the quick math. Didn't know there's such a package, tried it and turns out that the formatting would also be applied to the names column...but i did finally manage to place the names to the left and numbers to the right, learnt from this link haha
https://www.dotnetperls.com/padding-go

Reply:
awesome